Added new section in Summer Camps for On-Site Summer Camp

doelleb opened the pull request:
Added new section at the top of the "Summer Camp" page with title, description, and picture from past years

doelleb committed:
Added new section in Summer Camps for On-Site Summer Camp

Mallen220 commented on SummerCamps.md:
```diff
@@ -99,25 +99,21 @@ hero_height: is-medium
-</ul>
 
-<h3><b>No experience is required!</b></h3>
+<h2 style="font-size: 45px; text-align: left; color:#5777a8;"> Penguin Coding School Summer Camp </h2>
```
Why does it still have this name? I thought the goal was to seperate from Penguin Coding School Summer Camp?

Also trim white space and use relative font sizes, not px. (See rem and em for reference. )

Mallen220 replied:
Also, why are we using html here? Is it using Markdown below? Switch these to be consistent. Please use md when viable in md files for readability.

Mallen220 commented on SummerCamps.md:
```diff
@@ -99,25 +99,21 @@ hero_height: is-medium
 
 <div class="centered">
```
Why are we centering here? Everything Looks left alligned.

Mallen220 commented on SummerCamps.md:
```diff
@@ -99,25 +99,21 @@ hero_height: is-medium
 
 <div class="centered">
     <p>
```
Why use a large \<p> tag when all the contents can be done easier with md? We are in a md file not an html file.

Mallen220 commented on SummerCamps.md:
```diff
@@ -99,25 +99,21 @@ hero_height: is-medium
-This summer, we are excited to announce our collaboration with both Penguin Coding School and The Robo Hub!
-The buttons below will send you to a PDF of a flyer with information for both the Penguin Coding School Robotics Camp and The Robo Hub Robotics Camp, both in partnership with the Brookline Robotics Initiative.
+Brookline Robotics Initiative offers on-site robotics camps as a unique employee benefit. For a tax-deductible donation of $5,000-$10,000, we bring our full camp program directly to your workplace—complete with materials, instructors, and curriculum.
+<br><br>Employees can register 1-3 children (grades 3-8) for hands-on robotics activities right at the office, solving summer childcare challenges while providing quality STEM education. We offer flexible formats including single-day, 3-day, or week-long camps, typically running 9am-4pm.
```
Switch this to markdown and remove \<br>

Mallen220 replied:
Are we opposed to longer than a week-long if offered?

Mallen220 commented on SummerCamps.md:
```diff
@@ -99,25 +99,21 @@ hero_height: is-medium
-The buttons below will send you to a PDF of a flyer with information for both the Penguin Coding School Robotics Camp and The Robo Hub Robotics Camp, both in partnership with the Brookline Robotics Initiative.
+Brookline Robotics Initiative offers on-site robotics camps as a unique employee benefit. For a tax-deductible donation of $5,000-$10,000, we bring our full camp program directly to your workplace—complete with materials, instructors, and curriculum.
+<br><br>Employees can register 1-3 children (grades 3-8) for hands-on robotics activities right at the office, solving summer childcare challenges while providing quality STEM education. We offer flexible formats including single-day, 3-day, or week-long camps, typically running 9am-4pm.
+<br><br>With 5 years of experience and a 98% satisfaction rating, our camps provide a convenient, impactful way to support your team while investing in the next generation of innovators.
```
See previous \<br> comment.

Mallen220 commented on SummerCamps.md:
```diff
@@ -99,25 +99,21 @@ hero_height: is-medium
+<br><br>Employees can register 1-3 children (grades 3-8) for hands-on robotics activities right at the office, solving summer childcare challenges while providing quality STEM education. We offer flexible formats including single-day, 3-day, or week-long camps, typically running 9am-4pm.
+<br><br>With 5 years of experience and a 98% satisfaction rating, our camps provide a convenient, impactful way to support your team while investing in the next generation of innovators.
+
+<br><br><b>Interested in bringing BRI to your company?</b> Contact us to learn more about this turnkey employee benefit program.
```
See previous \<br> comment.
In md you can do \*\***Bolded text here**** to bold text. Replace the html \<b> tags with this.

Mallen220 commented on SummerCamps.md:
```diff
@@ -99,25 +99,21 @@ hero_height: is-medium
+<br><br><b>Interested in bringing BRI to your company?</b> Contact us to learn more about this turnkey employee benefit program.
+
 </p>
 </div>
```
\</p>
\</div>
Can both be removed and converted to a md block.

Mallen220 commented on images/Mentors/MichelleMcGone.png:
This file should not be deleted. It introduces a breaking change on the Meet The Team page.

Mallen220 commented on images/SummerCampFlyers/OnSiteSummerCamp.png:
Why are we adding this image? Also, it's not our image, so it shouldn't be on our website without proper credit.

Mallen220 committed:
Add Georgie, Yael, and Neta team member images

Added new images for team members Georgie and Neta. Updated Yael's image with a new version.

Mallen220 committed:
Revert "Add Georgie, Yael, and Neta team member images"

This reverts commit c61f2ac35c05dbd1eb96e4a205f8cf947094b633.

doelleb committed:
Changed SummerCamps.md to mostly markdown

doelleb commented:
Fixed comments from previous PR, removed image and changed to mostly markdown.
